configure.ac

```diff
@@ -37,14 +37,23 @@
 #
 
 AC_PREREQ([2.63])
-AC_INIT([pbspro], [18.1.0], [pbssupport@altair.com])
+# Use PBS_VERSION to override the version statically defined here. For example:
+# ./configure PBS_VERSION=19.1.2 --prefix=/opt/pbs
+AC_INIT([PBS Professional],
+  [19.0.0],
```

Comment:
Is it possible to get this value from pbspro.spec? that way we have to change PBS version only one place when we bump up version.

Reply:
I was attempting to consolidate the version numbers in configure.ac and pbspro.spec, but the fact that they can be defined as parameters to configure and rpmbuild made that impossible. Instead, I updated the spec file to obtain the version number from the source tarball that "make dist" creates. It's an improvement, just not to the extent I was hoping. I'm open to any suggestions you or others may have.

Reply:
Oh yes! I totally forget, now with your changes, pbspro.spec will try to find pbs_version from tarball that make dist creates and that tarball will get version from this place so ultimately pbspro.spec will get version from this place only.

Just a thought: can we use configure substitute add version in pbspro.spec? like we are doing for test/fw/ptl/__init__.py.in and test/fw/setup.py.in,
Because anyways we have to run `configure` and `make dist` to rpmbuild and pbspro.spec

Reply:
That might just work. Thanks for the suggestion, I'll give it a try!